android/app/src/main/java/org/electroncash/electroncash3/ColdLoad.kt

```diff
@@ -0,0 +1,86 @@
+package org.electroncash.electroncash3
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Intent
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.Toast
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.ViewModel
+import com.google.zxing.integration.android.IntentIntegrator
+import kotlinx.android.synthetic.main.load.*
+import kotlinx.android.synthetic.main.main.*
+
+// This provides a dialog to allow users to input a string, which is then broadcast
+// on the bitcoin cash network. Strings are not validated, but broadcast_transaction2 should throw error which is toasted.
+// Valid transaction quickly show up in transactions.
```

Comment:
Please keep lines to less than 100 characters if possible, otherwise it's difficult to work with two files side by side.

Reply:
fixed